conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml

```diff
@@ -0,0 +1,20 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  ocs_operator_nodes_to_label: 6
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 6
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
+  ocs_operator_nodes_to_label: 6
```

Comment:
I guess this is not right section and should be only on line 4 in DEPLOYMENT section.

Reply:
Add # TODO: Add deployment polarion ID